Dockerfile

```diff
@@ -44,10 +44,12 @@ RUN \
 # unzip the zip file into /go-server, after stripping the first path prefix
   unzip /tmp/go-server.zip -d / && \
   rm /tmp/go-server.zip && \
-  mv go-server-17.11.0 /go-server
+  mv go-server-17.11.0 /go-server && \
+  mkdir -p /docker-entrypoint.d
 
 COPY logback-include.xml /go-server/config/logback-include.xml
 
 ADD docker-entrypoint.sh /
+ADD install_plugins.sh /docker-entrypoint.d/install_plugins.sh
```

Comment:
You should not touch this file. It's generated automatically via a build script.